app/models/course_user.rb

```diff
@@ -215,6 +215,7 @@ def latest_learning_rate_record
   def set_defaults # :nodoc:
     self.name ||= user.name if user
     self.role ||= :student
+    self.timeline_algorithm ||= course.default_timeline_algorithm if course
```

Comment:
Does not seem to work. Can just remove this as of now.

Reply:
OK